src/models/emotion_detection/enhanced_bert_classifier.py

```
@@ -0,0 +1,534 @@
+        try:
+            # Get BERT outputs
+            bert_outputs = self.bert(input_ids=input_ids, attention_mask=attention_mask)
+            pooled_output = bert_outputs.pooler_output
+
+            # Classification head
+            logits = self.classifier(pooled_output)
+            
+            # Apply temperature scaling
+            logits = logits / self.temperature
+            
+            return logits
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)